stories/10-For Developers/01 - Best Practices.stories.mdx

```diff
@@ -37,7 +37,7 @@ Here's a example of a button that has `secondary` and `large` variants.
 	border: none;
 	color: rgb(var(--color-text-inverse));
 	padding: var(--space-3);
-	font-size: var--font-size-body-1);
+	font-size: var--font-size-5);
```

Comment:
can you clean up this typo? (originally mine, thanks)

Reply:
can't believe i didn't see that!